### DISCUSSION TITLE
IOS电脑端无授权码

### DISCUSSION THREAD

**Discussion by sabrinareng** [getlantern/lantern-forum-cn#254] [Q&A]
跟论坛其他几位问题一样，在ios端上找不到账号的授权验证码
<img width="682" alt="截屏2024-03-18 下午10 33 53" src="https://github.com/getlantern/lantern-forum-cn/assets/163874792/ff82bf3f-4290-462c-8f56-39057f62751e">

账号是：ssr789@163.com

**Comment by MaxMKMax** [comment]
@sabrinareng 您好。好的，我会帮您转达给我们的负责人。抱歉，他们将于今晚按照美国时区上线。我现在就通知他们，并帮助您今晚再次和他们跟进。会尽快帮您解决。抱歉让您久等了。

您还可以帮忙查看您的电子邮件，他们会手动向您发送验证码。谢谢!

**Comment by sabrinareng** [comment]
付款凭证：

![Uploading IMG_7391.PNG…]()



请尽快发送验证码给我，感谢～

**Reply by sabrinareng** [reply]
不知道能不能看到上传的截图，我这里怎么都点不开

![IMG_7391](https://github.com/getlantern/lantern-forum-cn/assets/163874792/c57dd5d6-eefb-462c-900e-a2483b845d9c)

**Reply by MaxMKMax** [reply]
@sabrinareng 您好. 请您稍等! 我们在研究是否能手动发出授权码. 或者通过别的方法. 不好意思. 请等等.

**Reply by sabrinareng** [reply]
您好，都一年了。我的问题依然没解决。。。 @Mike777Khoo

**Reply by MaxMKMax** [reply]
@sabrinareng 您好, 最新版本应该已经解决这个问题了. 请您更新看看. 同时您也提供我您的账号邮箱, 我看客服那里可以帮到您什么吗.
